src/PrestaShopBundle/Form/Admin/Sell/Product/HeaderType.php

```diff
@@ -92,11 +100,14 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     'data-combinations-warning' => $this->trans('This will delete all combinations.', 'Admin.Catalog.Notification'),
                     'data-pack-warning' => $this->trans('This will delete the list of products in this pack.', 'Admin.Catalog.Notification'),
                     'data-virtual-warning' => $this->trans('This will delete the associated virtual file.', 'Admin.Catalog.Notification'),
+                    'data-ecotax-warning' => $this->trans('This will reset the ecotax value and may impact your retail price tax included.', 'Admin.Catalog.Notification'),
```

Comment:
```suggestion
                    'data-ecotax-warning' => $this->trans('This will reset the ecotax value and may impact your retail price (tax incl.).', 'Admin.Catalog.Notification'),
```

Reply:
Thanks @Julievrz it's fixed